sklearn/base.py

```diff
@@ -436,15 +436,15 @@ def fit_predict(self, X, y=None):
 
         Parameters
         ----------
-        X : ndarray, shape (n_samples, n_features)
+        X : ndarray of shape (n_samples, n_features)
             Input data.
 
-        y : Ignored
+        y : Ignored, default=None
```

Comment:
I would revert this for now. The current standard in the library does not denote the type for y if it is ignored.